source/draw/RasterImage.ooc

```diff
@@ -57,6 +57,13 @@ RasterImage: abstract class extends Image {
 //		TODO: The stuff
 		result
 	}
+	save: virtual func (filename: String) -> Int { Debug raise("RasterImage save unimplemented for format!"); 0 }
+	kean_draw_rasterImage_getStride: unmangled func -> UInt { this stride }
+	kean_draw_rasterImage_save: unmangled func (path: const Char*) {
+		pathString := String new(path)
+		this save(pathString)
+		pathString free()
+	}
```

Comment:
Unmangled functions at the end of the class, we said. Even below static methods? I think so. You disagree?